scripts/error_codes.py

```diff
@@ -188,6 +188,7 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
     # Warning (1878): SPDX license identifier not provided in source file. ....
     # Warning (3420): Source file does not specify required compiler version!
     test_ids |= find_ids_in_cmdline_test_err(path.join(top_dir, "test", "cmdlineTests", "error_codes", "err"))
+    test_ids |= find_ids_in_cmdline_test_err(path.join(top_dir, "test", "cmdlineTests", "yul_unimplemented", "err"))
```

Comment:
@a3d4 any reason why we do not just check all commandline tests?

Reply:
Was it intentional to remove `error_codes`?

Reply:
In both lines, this is the path to a specific commandline test. error_codes is just another test, and `yul_unimplemented` is the one I added.

Reply:
> @a3d4 any reason why we do not just check all commandline tests?

I wanted to have error codes covered by regular tests, wherever possible/reasonable, and fall back to the command line tests or boost tests otherwise.